exovetter/utils.py

```diff
@@ -0,0 +1,13 @@
+import typing
```

Comment:
I don't know how I feel about applying typing here. I have no plans to start using it for this package. But I guess we can leave it in though it does make the code a little harder to read.

Reply:
I'm not attached to type hints. I don't mind omitting them